app/src/main/java/com/example/parkingsystemjava/mvp/presenter/DateDialogPresenter.java

```diff
@@ -0,0 +1,4 @@
+package com.example.parkingsystemjava.mvp.presenter;
+
+public class DateDialogPresenter {
```

Comment:
this class must to match with the current fragment, so change to: DateReservationDialogPresenter

Reply:
done